src/lib/Components/Artist/Articles/Article.tsx

```diff
@@ -1,10 +1,10 @@
-import * as React from "react"
+import React from "react"
```

Comment:
This looks nicer to me – what's the difference? Is there any rationale to keep the old way?

Reply:
Argh, just re-read your PR description, got it 👍

Reply:
:)